package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "cmc-citizen-frontend",
-  "version": "1.1.1",
+  "version": "1.2.0",
```

Comment:
why are you doing this?

Reply:
I bumped because we have new code that will be deployed during next release and incrementing the feature version made more sense to me. It's probably unrelated to this particular PR, I can remove this.

Reply:
please remove, its unrelated